client/settings/express-checkout/test/index.test.js

```diff
@@ -169,7 +169,7 @@ describe( 'ExpressCheckout', () => {
 				<ExpressCheckout />
 			</WCPaySettingsContext.Provider>
 		);
-		const linkCheckbox = container.getByLabelText( 'Stripe Link' );
+		const linkCheckbox = container.getByLabelText( 'Link by Stripe' );
```

Comment:
In case we decide to add translation tags per [this comment](https://github.com/Automattic/woocommerce-payments/pull/4852/files#r998005780), we may need to include it in the `Link by Stripe` instances on this file too.

Reply:
we don't need to include translations in tests. should work with the base text.